diff --git i/Python27/bme280_sample.py w/Python27/bme280_sample.py
index 3447ecf..6b1a5fe 100644
--- i/Python27/bme280_sample.py
+++ w/Python27/bme280_sample.py
@@ -92,7 +92,7 @@ def compensate_P(adc_P):
 	v2 = ((pressure / 4.0) * digP[7]) / 8192.0
 	pressure = pressure + ((v1 + v2 + digP[6]) / 16.0)  
 
-	print "pressure : %7.2f hPa" % (pressure/100)
+	print('{ "pressure": %-7.2f }' % (pressure/100))
 
 def compensate_T(adc_T):
 	global t_fine
@@ -100,7 +100,7 @@ def compensate_T(adc_T):
 	v2 = (adc_T / 131072.0 - digT[0] / 8192.0) * (adc_T / 131072.0 - digT[0] / 8192.0) * digT[2]
 	t_fine = v1 + v2
 	temperature = t_fine / 5120.0
-	print "temp : %-6.2f ℃" % (temperature) 
+	print('{ "temperature": %-7.2f }' % (temperature))
 
 def compensate_H(adc_H):
 	global t_fine
@@ -114,7 +114,7 @@ def compensate_H(adc_H):
 		var_h = 100.0
 	elif var_h < 0.0:
 		var_h = 0.0
-	print "hum : %6.2f ％" % (var_h)
+	print('{ "humidity": %-7.2f }' % (var_h))
 
 
 def setup():
